pandas/core/series.py

```diff
@@ -2138,6 +2138,14 @@ def map(self, arg, na_action=None):
         two   bar
         three baz
 
+        >>> z
```

Comment:
Could you add a single sentence between the examples? Eg something like "Mapping a dict works similarly to mapping a Series (the dict keys are mapped on the index labels): "